stdlib/ctypes/_endian.pyi

```diff
@@ -1,11 +1,15 @@
 import sys
-from _ctypes import RTLD_GLOBAL as RTLD_GLOBAL, RTLD_LOCAL as RTLD_LOCAL, Structure, Union
-from ctypes import DEFAULT_MODE as DEFAULT_MODE, cdll as cdll, pydll as pydll, pythonapi as pythonapi
 
-if sys.version_info >= (3, 12):
+if sys.version_info >= (3, 13):
+    from ctypes import Array as Array, Structure, Union
```

Comment:
Why add `Array` as new re-export?

Reply:
It's exported in the implementation: https://github.com/python/cpython/blob/094375b9b7e087a4f0f60541dc7f2dc53be92646/Lib/ctypes/_endian.py#L2

Reply:
It's just imported there. This is a private module, and it doesn't seem like it's intentionally re-exported.

Reply:
I see, was trying to match the runtime as closely as possible, but makes sense.

Reply:
https://github.com/python/typeshed/pull/12372/commits/2074c3da25d04aec81d9029760959c0f7b57b717